### ISSUE TITLE
why queryParams is undefined in server?

### ISSUE THREAD

**Issue by Goluis** [kadirahq/flow-router#237]
```js
/blog?skip=10
FlowRouter.route('/blog', {
  subscriptions: function (params, queryParams) {
    console.log(queryParams) // --> client: {skip: 10} - server: undefined
  }
});
```
before I go to Kadira: flow-router everything worked fine

**Comment by arunoda** [comment]
I hope this with FlowRouter SSR?

**Comment by arunoda** [comment]
Yep. It's a bug.
https://github.com/kadirahq/flow-router/blob/ssr/server/route.js#L39

**Comment by abernix** [comment]
This isn't just with SSR (although the bug you linked also exists).  This happens if you have FastRender enabled on current version `2.0.1` of `kadira:flow-router`.

https://github.com/kadirahq/flow-router/blob/master/server/plugins/fast_render.js#L28

**Comment by arunoda** [comment]
For that, I think we need a quick fix.
Will work on that.

**Comment by arunoda** [comment]
Okay. I just fixed this and did a release: `2.0.2`

**Comment by abernix** [comment]
Awesome.  Works great.  Thanks for the quick-fix.   Assuming you'll leave this open for SSR?

**Comment by arunoda** [comment]
Yep.

**Comment by ashokgelal** [comment]
Has this been fixed for SSR? When using queryParams with ReactLayout, I get an "Invalid Checksum" warning. I did some digging and turned out params are not available on server.

**Comment by arunoda** [comment]
It's not fixed yet. Will be soon. Within few days.
We are looking for a beta release for React soon.

**Comment by ashokgelal** [comment]
Any updates on this issue?

**Comment by arunoda** [comment]
Check now. Published a new version: `v3.3.0`
